src/test/java/tests/LoginTests.java

```diff
@@ -73,12 +73,12 @@ public void tc06_AdditionalPagesDisplayOnceUserLoggedIn() {
         myTicketPage.goToMyTicketPage();
 
         LogHelper.info("Check if the user may be redirected to the My Ticket page");
-        Assert.assertEquals(DriverHelper.getCurrentTitle(), "Safe Railway - My Ticket", "The user is not redirected to the My Ticket page");
+        Assert.assertEquals(DriverUtils.getCurrentUrl(), "http://www.railway2.somee.com/Page/ManageTicket.cshtml", "The user is not redirected to the My Ticket page");
```

Comment:
do you consider about create variables for these URL?

Reply:
I created variables for these URLs. Help me check it